app/src/main/java/edu/ucsc/cmps115_spring2017/face2name/Layer/LayerView.java

```diff
@@ -0,0 +1,62 @@
+package edu.ucsc.cmps115_spring2017.face2name.Layer;
+
+/**
+ * Created by Chris Myau on 4/25/2017.
+ */
+
+import android.content.Context;
+
+import android.util.AttributeSet;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.View;
+import 	android.view.MotionEvent;
+
+
+public final class LayerView extends View
+{
+    private Paint paint;
+    int color;
+    int param1, param2, param3, param4;
+    public LayerView(Context context) {
+        super(context);
+    }
+    public LayerView(Context context, AttributeSet set) {
+        super(context, set);
+        paint = new Paint();
+        color =  Color.parseColor("#e60000");
```

Comment:
The above two lines need to be in every constructor. Consider putting them in an initialization block to avoid copying them into every constructor.